content_schemas/formats/shared/definitions/_whitehall.jsonnet

```diff
@@ -35,6 +35,11 @@
         type: "string",
         format: "uri",
       },
+      medium_resolution_url: {
```

Comment:
I had a chat with some frontendy people - Kuba, Max - and they clearly  expressed desire to provide srcset attribute in the ui so that different devices could choose the most appropriate image size. For example the Coronation page was difficult to make look nice because the image in its small resolution looked pixelated. 

I think this is useful for all types because this way in the future, all types could publish at least 3 sizes, and then rendering world could utilize those 3 sizes to create a srcset attribute.

Please see this pr for an example: https://github.com/alphagov/collections/pull/3426